gfx.cpp

```diff
@@ -8882,6 +8884,11 @@ class GfxInternal
         shader_args.push_back(L"-I"); shader_args.push_back(L".");
         shader_args.push_back(L"-T"); shader_args.push_back(wshader_profile.data());
         shader_args.push_back(L"-HV 2021");
+        shader_args.push_back(L"-Wno-parameter-usage");
+        shader_args.push_back(L"-Wno-uninitialized");
+        shader_args.push_back(L"-Wno-conditional-uninitialized");
+        shader_args.push_back(L"-Wno-sometimes-uninitialized");
+        shader_args.push_back(L"-fdiagnostics-format=msvc");
```

Comment:
Should these be added only when `debug_shaders_` is enabled? (assuming this is part of the workaround)

Reply:
Possibly. I havnt actually tested whether they are needed for release builds as there is a lot of permutations to go through (they disable diagnostics so I figured thees no harm).
These are all inherited from the dxc-6.8 branch though and the only new one added for this PR is `shader_args.push_back(L"-fdiagnostics-format=msvc");` which is required all the time to get the compiler output in the correct format

Reply:
Sounds good, let's keep it as is then.